client/src/test/scala/org/http4s/client/ClientRouteTestBattery.scala

```diff
@@ -56,93 +50,95 @@ abstract class ClientRouteTestBattery(name: String)
       }
     }
 
-  withResource(JettyScaffold[IO](1, false, testServlet)) { jetty =>
-    withResource(clientResource) { client =>
-      val address = jetty.addresses.head
-
-      Fragments.foreach(GetRoutes.getPaths.toSeq) { case (path, expected) =>
-        s"Execute GET: $path" in {
-          val name = address.getHostName
-          val port = address.getPort
-          val req = Request[IO](uri = Uri.fromString(s"http://$name:$port$path").yolo)
-          client
-            .run(req)
-            .use(resp => checkResponse(resp, expected))
-            .unsafeRunTimed(timeout)
-            .get
-        }
-      }
+  def clientFixture: FunFixture[(JettyScaffold, Client[IO])] =
+    ResourceFixture(((JettyScaffold[IO](1, false, testServlet), clientResource)).tupled)
 
-      name should {
-        "Strip fragments from URI" in {
-          skipped("Can only reproduce against external resource.  Help wanted.")
-          val uri = Uri.uri("https://en.wikipedia.org/wiki/Buckethead_discography#Studio_albums")
-          val body = client.run(Request[IO](uri = uri)).use(e => IO.pure(e.status))
-          body must returnValue(Ok)
-        }
+  // Need to override the context shift from munitCatsEffect
+  // This is only required for JettyClient
+  implicit val contextShift: ContextShift[IO] = Http4sSuite.TestContextShift
 
-        "Repeat a simple request" in {
-          val path = GetRoutes.SimplePath
+  clientFixture.test("Strip fragments from URI".ignore) { case (_, client) =>
```

Comment:
I'm not totally sure why was this ignored in the first place

Reply:
I don't know what's external about it.  It doesn't look like a good test.  A request-target with a fragment is illegal, so we'd want to be sure that the request line submitted by a client has no fragment.  Different servers have different tolerances for it.  Jetty allows it and Tomcat forbids it, for instance.  Reading directly from a socket to see what's sent would be the most reliable way to test it. but a pain.

Reply:
Maybe I should remove it then